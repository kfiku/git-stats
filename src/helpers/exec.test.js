// @ts-check

const exec = require('./exec')

/* eslint-disable fp/no-unused-expression */
/* eslint-disable fp/no-nil */

describe('exec', () => {
  it('should work fine', async () => {
    const result = await exec(__dirname, 'ls exec.js')
    expect(result).toMatchSnapshot()
  })

  it('should throw', async () => {
    const fn = async () => {
      await exec(__dirname, 'NonExistingCmd')
    }

    expect(fn()).rejects.toBeDefined()
  })
})
