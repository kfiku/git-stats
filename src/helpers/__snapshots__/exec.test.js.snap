// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exec should work fine 1`] = `
"exec.js
"
`;
